backend/config/sync/core-store.plugin_content_manager_configuration_content_types##api##support.support.json

```diff
@@ -206,59 +206,67 @@
       }
     },
     "layouts": {
-      "list": [
-        "id",
-        "FullName",
-        "Email",
-        "Phone"
-      ],
       "edit": [
         [
           {
             "name": "FullName",
-            "size": 6
-          },
+            "size": 12
+          }
+        ],
+        [
           {
             "name": "Email",
-            "size": 6
+            "size": 12
           }
         ],
         [
           {
             "name": "Phone",
-            "size": 6
-          },
+            "size": 12
+          }
+        ],
+        [
           {
             "name": "Title",
-            "size": 6
+            "size": 12
           }
         ],
         [
           {
             "name": "Message",
-            "size": 6
-          },
+            "size": 12
+          }
+        ],
+        [
           {
             "name": "DateSubmitted",
```

Comment:
Do you think DateSubmitted is needed?

Reply:
Yes. It will allow the admin to see when they received the ticket

Reply:
I agree